### ISSUE TITLE
GCRCatalogs cannot load quantities

### ISSUE THREAD

**Issue by danielsf** [LSSTDESC/gcr-catalogs#20]
The following code:
```
import GCRCatalogs
cat=GCRCatalogs.load_catalog('proto-dc2_v2.0')
qties = cat.get_quantities(['otherLuminosities/diskLuminositiesStellar:V:rest'])
```
gives the following error
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 416, in get_quantities
    d = self._concatenate_quantities(quantities, pre_filters, post_filters)
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 370, in _concatenate_quantities
    for data in self._get_quantities_iter(quantities, pre_filters, post_filters):
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 356, in _get_quantities_iter
    data.update(self._load_quantities(rest_quantities.difference(set(data)), dataset))
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 301, in _load_quantities
    native_data = {q: self._fetch_native_quantity(dataset, q) for q in self._translate_quantities(quantities)}
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 301, in <dictcomp>
    native_data = {q: self._fetch_native_quantity(dataset, q) for q in self._translate_quantities(quantities)}
  File "/global/homes/d/danielsf/generic-catalog-reader/GCR.py", line 442, in _fetch_native_quantity
    return np.asanyarray(dataset[native_quantity]).ravel()
TypeError: 'function' object is not subscriptable
```

**Comment by yymao** [comment]
@danielsf can you try `pip install GCR==0.6.1`?

**Comment by danielsf** [comment]
Okay.  That worked.  Thanks.

I was working with clone of generic-catalog-reader, is that no longer the right thing to do (well, I guess it is obviously not the right thing to do; maybe I am asking "did we know that would break")?

**Comment by yymao** [comment]
@danielsf yes, that was known and fixed in #18. Should have given you a heads-up, sorry! And later I also made GCRCatalogs's minor version in sync with GCR's minor version. Both are now at 0.6.x
